### PR TITLE
Kiwi import test

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@hennevogel please review!

**Comment by ChrisBr** [comment]
@hennevogel ping ... it''s almost two weeks old!

File: src/api/spec/models/package_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Service, vcr: true do
```
Comment (mdeniz) [inline src/api/spec/models/package_spec.rb]: You are describing "Service" or "Package"??

Comment (bgeuken) [inline src/api/spec/models/package_spec.rb]: +1 for Package

File: src/api/spec/models/package_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  let(:home_project) { Project.find_by(name: user.home_project_name) }
+  let(:package) { create(:package, name: 'test_package', project: home_project) }
+  let(:services) { package.services }
+
+  context '#save_file' do
+    before do
+      User.current = user
+    end
+
+    it 'calls #addKiwiImport if filename ends with kiwi.txz' do
```
Comment (bgeuken) [inline src/api/spec/models/package_spec.rb]: Would a test for not calling addKiwiImport in all other cases make sense here?

Comment (ChrisBr) [inline src/api/spec/models/package_spec.rb]: Hm, okay!

File: src/api/spec/models/package_spec.rb (hunk 1 above)
Comment (ChrisBr) [inline src/api/spec/models/package_spec.rb]: Agreed!

**Comment by ChrisBr** [comment]
@bgeuken @mdeniz fixed your suggestions!

**Comment by bgeuken** [comment]
LGTM

**Comment by mdeniz** [comment]
LGTM :+1:

**Pull request merged**